components/youtube_video/youtube_video.jsx

```diff
@@ -69,16 +69,16 @@ export default class YoutubeVideo extends React.PureComponent {
     }
 
     handleYoutubeTime(link) {
-        const timeRegex = /[\\?&]t=([0-9]+h)?([0-9]+m)?([0-9]+s?)/;
+        const timeRegex = /[\\?&](t|start|time_continue)=([0-9]+h)?([0-9]+m)?([0-9]+s?)/;
```

Comment:
This doesn't need to be changed, but if you add `?:` to the start of the brackets like `(?:t|start|time_continue)`, it makes the group non-capturing so that it doesn't change the indices below.